.github/workflows/cron-update-icons.yml

```diff
@@ -42,17 +42,25 @@ jobs:
         env:
           FIGMA_TOKEN: ${{ secrets.FIGMA_TOKEN }}
 
+      - uses: actions/checkout@v4
+
       - name: Create PR
-        uses: actions/github-script@v6
+        uses: peter-evans/create-pull-request@v6
```

Comment:
We used to have peter-evans but we changed to the official github actions one. I think we should try to keep it, if it is possible 🤔